lib/attrsets.nix

```diff
@@ -431,7 +432,7 @@ rec {
        => { x = "x-foo"; y = "y-bar"; }
 
      Type:
-       mapAttrs :: (String -> Any -> Any) -> AttrSet -> AttrSet
+       mapAttrs :: (name :: string -> value :: any -> any ) -> AttrSet -> AttrSet
```

Comment:
Yep had the same thought. I will remove names then from positional arguments.
Still like the names in AttrSet.

Reply:
Yeah names for attribute sets are good because they're part of the type, they need to be known by the caller. In contrast, names for positional arguments don't influence the type.